core/imageroot/usr/local/agent/actions/get-status/20read

```diff
@@ -41,12 +41,28 @@ status["ui_name"] = rdb.get(f'module/{os.environ["MODULE_ID"]}/ui_name') or ""
 status["node"] = rdb.hget(f'module/{os.environ["MODULE_ID"]}/environment', 'NODE_ID')
 status["node_ui_name"] = rdb.get(f'node/{status["node"]}/ui_name') or ""
 
+rootfull = os.getuid() == 0
+image_filter = []
+
+if rootfull:
+    systemd_path = f'/etc/systemd/system/{status["instance"]}*.service'
+    # prepare image filter
+    env = rdb.hgetall(f'module/{status["instance"]}/environment')
```

Comment:
To improve performance and consistency, consider to read the environment from the local filesystem, instead of querying Redis (that is just a remote copy)

```suggestion
    env = agent.read_envfile(os.environ["AGENT_STATE_DIR"] + "/environment")
```

Also, `status["node"]` (line 42) can be set to `env['NODE_ID']` too